### ISSUE TITLE
stream claims with no content type are served as jpegs by default

### ISSUE THREAD

**Issue by kauffj** [lbryio/spee.ch#31]
[https://spee.ch/cow-air-balloon](https://spee.ch/cow-air-balloon) triggers a file download, rather than being served. Likely related to the headers being sent, but I did not dig very far.

**Comment by billbitt** [comment]
I traced the issue to the fact that some assets do not have a content type and/or do not have a file extension.  I updated the code to send any unknown files as jpegs.  This appears to have solved the issue, as I now get the image returned & displayed in chrome, IE, and postman.   However, the better long term solution is to decipher the file type from the file itself and append the correct file extension & headers.  @filipnyquist has expressed interest in handling that preferred fix.  I am changing this to P3 since the default-to-jpeg solution is working for now.
